sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -82,6 +85,125 @@ With this configuration, the Azure Monitor Distro will use the credentials of th
 
 Note that the `Credential` property is optional. If it is not set, Azure Monitor Distro will use the Instrumentation Key from the Connection String to send data to Azure Monitor.
 
+### What is Included in the Distro
```

Comment:
This write up is good, but it's in the "Getting Started" section.
I think this belongs earlier in the document, I suggest before line 5.